src/main/java/org/jetbrains/bsp/bazel/server/BazelBspServerConfig.java

```diff
@@ -23,7 +23,7 @@ public static BazelBspServerConfig from(String[] args) {
     List<String> targetProjectPaths =
         args.length == 2
             ? Arrays.asList(args[1].split(","))
-            : Collections.unmodifiableList(Collections.singletonList("//..."));
+            : Collections.singletonList("//...");
```

Comment:
Any reason for this change to be happening here? Not sure if its under the scope of the PR or if it should be happening as a whole

Reply:
yea, I isn't connected with this PR - intellij marked it and I decided to change it - I can revert it